content/get-involved.md

```diff
@@ -11,7 +11,7 @@ In order to understand the project better and come up with reasonable solutions,
 
 * Go through some Jaeger tutorials, such as [this blog post](https://medium.com/jaegertracing/take-jaeger-for-a-hotrod-ride-233cf43e46c2) or [this video](https://youtu.be/s7IrYt1igSM?si=B3NI6ruohKfSPUCl&t=445).
 * Run the [HotROD demo yourself](https://github.com/jaegertracing/jaeger/blob/main/examples/hotrod/README.md). The blogs and videos may be outdated, it's good to get hands on.
-* Review the [Jaeger architecture](/docs/latest/architecture/) and understand the components.
+* Review the [Jaeger architecture](</docs/{{% param latestV2 %}}/architecture/>) and understand the components.
```

Comment:
what do `<>` brackets do in this context?

Reply:
It is Markdown syntax that ensures that the Markdown processor sees the entire argument as a link reference, even if there are `{{...}}` inside. It isn't strictly necessary atm, but if ever the site starts using Prettifier, it will be necessary.